netdata-installer.sh

```diff
@@ -502,21 +507,23 @@ BANNER4
 fi
 
 have_autotools=
+# shellcheck disable=SC2046,SC2086
```

Comment:
Please put these on the individual lines that are triggering the warnings (517 for SC2046 and 518 for SC2086). 

We ideally want all of our overrides scoped as narrowly as possible.